learner_pathway_progress/management/commands/update_existing_pathways_progress.py

```diff
@@ -0,0 +1,27 @@
+"""
+Management command to update existing learner progress in pathway.
+"""
+from textwrap import dedent
+
+from django.core.management import BaseCommand
+
+from learner_pathway_progress.utilities import update_existing_pathways_progress
+
+
+class Command(BaseCommand):
+    """
+    Command to update learner progress in pathway.
+
+    Update pathway progress for all enterprise learners who have any course enrollments.
+
+    Examples:
+
+        ./manage.py lms update_existing_pathways_progress
+    """
+    help = dedent(__doc__)
+
```

Comment:
I think this command can be very helpful in the future if we want to update pathway progress for a single enterprise learner. Do you think if that would be possible to update progress of one specific learner through this command?

Reply:
Do you have any case in mind where we need to update the pathway progress for a single learner? Yes, it'll be possible to update the pathway progress for one learner by making a small change in the case.

Reply:
Like for instance if a bug shows up where a pathway progress for one specific learner isn't reflecting correctly on learner portal, we can run this management command to update it. But if we are planning to run this command periodically then we can just ignore what I said :)

Reply:
I think that bug shouldn't come in the first place as we emit the signals and change the pathway progress of the learner when he/she meets certain criteria.
This command will run only once to fix the existing data where the signal was calling before the data was saved and cause the issue. 
If we need to run it in future for a single learner, we can pass the learner id to this command with a small change in the code and it'll work.